docs/getting-started/installation-and-upgrade/resources/upgrade-cert-manager.md

```diff
@@ -259,13 +259,6 @@ cert-manager-webhook-787858fcdb-nlzsq      1/1     Running   0          2m
 
 ## Cert-Manager API change and data migration
 
----
-_New in v2.6.4_
-
-Rancher now supports cert-manager versions 1.6.2 and 1.7.1. We recommend v1.7.x because v 1.6.x will reach end-of-life on March 30, 2022. To read more, see the [cert-manager docs](../../../pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster.md#4-install-cert-manager). For instructions on upgrading cert-manager from version 1.5 to 1.6, see the upstream cert-manager documentation [here](https://cert-manager.io/docs/installation/upgrading/upgrading-1.5-1.6/). For instructions on upgrading cert-manager from version 1.6 to 1.7, see the upstream cert-manager documentation [here](https://cert-manager.io/docs/installation/upgrading/upgrading-1.6-1.7/).
-
----
-
```

Comment:
Same as above, lots of people still on early 2.6's.

Reply:
I'm going to merge this into the release branch, then create a new working branch to address this problem with the notes. The way I'm currently updating the PR makes it awkward to edit multiple pages at once.